hugo/content/citation/_index.md

```diff
@@ -25,31 +27,36 @@ You can also cite a specific species page from the Genome Portal. The easiest wa
 The general pattern for citing a species page can also be filled out using the following template:
 
 **In-text citation (APA format)**:\
-The [_Species name_] page in the Swedish Reference Genome Portal (access date).
+{{< citation_block >}}
+The <span class="highlight">[_Species name_]</span> page in the Swedish Reference Genome Portal (DATE_ACCESSED).
+{{< /citation_block >}}
 
-where [_Species name_] is a placeholder that should be replaced with the binomial name of the species in italics, and Swedish Reference Genome Portal (access date) is the APA formatted in-text pointer to the reference stated above in [Section 1.1](#11-citing-the-genome-portal).
+where [_Species name_] is a placeholder that should be replaced with the binomial name of the species in italics, and Swedish Reference Genome Portal with the access date provided is the APA formatted in-text pointer to the reference stated above in [Section 1.1](#11-citing-the-genome-portal).
 
 #### 1.3. In-text citation in the Data Availability Statement of a manuscript
 
 Researchers who have submitted their data to the Genome Portal may want to include the following text in the Data Availability statement in their research manuscript(s).
 
-"Visualisations of the genome assembly and annotation tracks from this publication can be found in the Swedish Reference Genome Portal (access date) at <https://genomes.scilifelab.se/[species_name]>."
+{{< citation_block >}}
+"Visualisations of the genome assembly and annotation tracks from this publication can be found in the Swedish Reference Genome Portal (DATE_ACCESSED) at <https://genomes.scilifelab.se/[species_name]>."
```

Comment:
```suggestion
"Visualisations of the genome assembly and annotation tracks from this publication can be found in the Swedish Reference Genome Portal (Retrieved DATE_ACCESSED) at <https://genomes.scilifelab.se/[species_name]>."
```

Reply:
Didn't add this to batch the first time by accident, but have now committed it.